### PR TITLE
Set up google places api to get stage latitude, longitude

### LINKED ISSUE #16: Extract location coordinates and description from GoogleAPI
1. Obtain Google API Credentials:

- Using Google Cloud Console, create API Key for project.
- Enable the following APIs: Places API and Geocoding API

2. Backend Implementation:

- Set up backend routes and controllers to handle API requests.
- Install the necessary packages to make HTTP requests in your backend.
- Store Google API credentials securely in your backend configuration (environment variables).

3. Fetch Location Coordinates (Places API):

- Implement a backend route to fetch place coordinates using the Places API.
- In the backend controller, make an HTTP GET request to the Places API endpoint with the place name and city
- Parse the response and extract the latitude and longitude coordinates.
- Return the coordinates as the API response.

4. Further response interests:

- Ratings of place
- Opening hours

### PR DESCRIPTION
**Pull request by vee-16**
Following function can be called from `server/googleapi/googleCoordinates.js`: 

 `getStageCoordinates` : 
 - input: 
    - StageLocationName and Destination
 - output: 
    - location object { lat: number, lng: number } or error
    - _Note:_ - if StageLocationName is a chain business, only first object from response is returned
 
To test `getStageCoordinates`:
 - `cd server`
 - `yarn coordinates:test`
 
 Further test:
 - taking any results after running `yarn coordinates:test`:
 
```
  ----- Coordinates for Stanley Park in Vancouver -----
  { lat: 49.30425839999999, lng: -123.1442522 }
```
 
 - pasting results in google maps
Screenshot 2023-07-13 at 2 40 38 AM

 - pasting object in apple maps
Screenshot 2023-07-13 at 2 50 30 AM

### CODE REVIEW

**Comment by vee-16** [comment]
closes #16

File: server/google_api/googleCoordinates.js
Comment (L0Lmaker) [inline server/google_api/googleCoordinates.js:1]: nit folder name: `googleapi` for consistency

File: server/google_api/googleCoordinates.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    const { location } = results.geometry
+    return location.lng
+  } catch (error) {
+    console.error('Error occurred in getStageLongitude:', error)
+    throw error
+  }
+}
+
+async function getStageRating(results) {
+  try {
```
Comment (L0Lmaker) [inline server/google_api/googleCoordinates.js]: Could we combine this into one function call that returns an object that looks like this:
```
{
  latitude: 10.0,
  longitude: 50.0
}
```

File: server/googleapi/googleCoordinates.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+const axios = require('axios')
+
+/**
+ Retrieves the coordinates of a stage location using Google Places API.
+ @param {string} destination - The destination of the stage.
+ @param {string} stageLocationName - The name of the stage location.
+ @returns {object} The coordinates of the stage location as an object { lat: number, lng: number }
+ @throws {Error} If there is an error while fetching place details.
+ */
+async function getStageCoordinates(destination, stageLocationName) {
```
Comment (L0Lmaker) [inline server/googleapi/googleCoordinates.js:11]: stage object? Lets keep it consistent to openai changes so theres no ambiguity in usage.

File: server/googleapi/googleCoordinates.js
Comment (L0Lmaker) [inline server/googleapi/googleCoordinates.js:1]: could you make a `test.js` file also so we can see the method in action?

File: server/googleapi/googleCoordinates.js (hunk 2 above)
Comment (vee-16) [reply server/googleapi/googleCoordinates.js:11]: so right now it returns an object as follows for example: {
  lat: 10.0,
  lng: 50.0
}

**Review by L0Lmaker — APPROVED** [review]
LGTM

**Review by AndyLiang1 — APPROVED** [review]

**Pull request merged**